examples/seismic/tutorials/fwi_gradient_marmousi.py

```diff
@@ -0,0 +1,118 @@
+import argparse
```

Comment:
This looks like it has a lot of common with `examples/seismic/inversion` it would be better to avoid duplications and update that example instead.